src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -137,6 +160,7 @@ void tabulate() throws TabulationCancelledException {
           runTabulationForConfig(config, castVoteRecords);
         }
       }
+      Logger.log(Level.INFO, "Tabulation session complete.  Results written to: %s", outputPath);
```

Comment:
double space -> single

Reply:
Why this?

Reply:
The double space after a period is a Satanic Practice.

Reply:
Yeah, double-spacing after a period is a relic of a bygone era. In any case, we use single spaces everywhere else.